frontend/src/components/events/EventList.js

```diff
@@ -1,35 +1,54 @@
 import React from "react";
 import EventSection from "./organisms/event_section";
 import "./index.css";
-import { findPastEndIndex } from "./utils";
+import { findFirstUpcomingIndex } from "./utils";
+import FeatureEvent from "./FeatureEvent";
 
 const EventList = ({ events }) => {
```

Comment:
Let's not implicitly assume ordering of events. Either order it in the API call or in the code here.